app/src/main/java/com/accountabilibuddies/accountabilibuddies/modal/Challenge.java

```diff
@@ -0,0 +1,106 @@
+package com.accountabilibuddies.accountabilibuddies.modal;
+
+import com.parse.ParseClassName;
+import com.parse.ParseObject;
+
+import java.util.Date;
+import java.util.List;
+
+@ParseClassName("Challenge")
+public class Challenge extends ParseObject {
+
+    //Default Parse columns
+    // objectId is a unique identifier for each saved object.
+    // createdAt and updatedAt represent the time that each object was created and last modified.
+    /**
+     Challenge {
+         long typeId -->Another table
+         String name;
+         String description;
+         String startDate;
+         String endDate;
+         int frequency; -->Another table
```

Comment:
Yup, but I still have to create those classes, they will come in a seperate pull request and will update these files. Too may modal classes will bloat up one PR request :)